Resources/Private/Partials/ContentElements/Media/Rendering/Image.html

```diff
@@ -3,10 +3,11 @@
     <f:for each="{variants}" key="name" as="variant">
         <f:variable name="breakpoint">{variant.breakpoint as integer}</f:variable>
         <f:variable name="width">{variant.width as integer}</f:variable>
+        <f:variable name="widthRetina">{variant.widthRetina as integer}</f:variable>
         <f:if condition="{width}">
-            <source data-variant="{name}" data-maxwidth="{width}" {f:if(condition: breakpoint, then: ' media="(min-width: {breakpoint}px)" ')}srcset="{f:uri.image(image: file, cropVariant: name, maxWidth: width)}">
+            <source data-variant="{name}" data-maxwidth="{width}" {f:if(condition: breakpoint, then: ' media="(min-width: {breakpoint}px)" ')}srcset="{f:uri.image(image: file, cropVariant: name, maxWidth: width)}{f:if(condition: widthRetina, then: ' , {f:uri.image(image: file, cropVariant: name, maxWidth: widthRetina)} 2x ')}">
         </f:if>
     </f:for>
     <img src="{f:uri.image(image: file, cropVariant: 'default', maxWidth: variants.default.width)}" title="{file.properties.title}" alt="{file.properties.alternative}">
 </picture>
-</html>
```

Comment:
Please ensure .editorconfig is loaded and an empty line is included at the end of each file.